server/bin/pbench-audit-server.py

```diff
@@ -307,7 +309,11 @@ def verify_results(rhier, verifylist):
         rhier.add_controller(controller)
 
         for dir_path in direct_entries:
-            path = Path(str(dir_path).split(os.path.join(controller, ""), 1)[1])
+            """ Here, we may encounter an additional subdirectory structure because
+                of "prefix" handling. The function `path_below_controller()' gives
+                us the path of that subtructure.
+            """
```

Comment:
This is more of a description for the method, and I think most of this paragraph belongs there.